frontend/src/app/views/evaluations/evaluations.component.ts

```diff
@@ -40,4 +43,47 @@ export class EvaluationsComponent implements OnInit {
     }
     return sum;
   }
+
+  @HostListener('document:keyup', ['$event'])
```

Comment:
Ten kawałek kodu powtórzony jest 3 razy. Skoro kandydat nie może wykonywać tych czynności w całej aplikacji to lepiej by było umieścić te host listenery w `AppComponent` - część z nich już tam jest na potrzeby zbierania metryk.